_docs/payout-details.md

```diff
@@ -399,44 +399,9 @@ The payment reference can also be provided in the recipient details hash optiona
 
 ## XOF::Cash
 
-For Senegalese cash pickup requests there are two payout provider options available:
+For Senegalese cash pickup requests we have the following payout provider options available:
```

Comment:
Can this be singular since we have just the one?
```suggestion
For Senegalese cash pickup requests we have the following payout provider option available:
```